test/functional/test_framework/wallet.py

```diff
@@ -208,14 +209,21 @@ def send_self_transfer_multi(self, **kwargs):
         return {'new_utxos': [self.get_utxo(txid=txid, vout=vout) for vout in range(len(tx.vout))],
                 'txid': txid, 'hex': tx.serialize().hex(), 'tx': tx}
 
-    def create_self_transfer_multi(self, *, from_node, utxos_to_spend=None, num_outputs=1, fee_per_output=1000):
+    def create_self_transfer_multi(
+            self, *, from_node,
+            utxos_to_spend: Optional[List[dict]] = None,
```

Comment:
Same comment, this doesn't seem required; could you mention that in the commit comment?